.travis.yml

```diff
@@ -0,0 +1,6 @@
+sudo: false
```

Comment:
Don't add `.travis.yml` in this PR, we use `webpack-defaults` for all repositories https://github.com/webpack-contrib/webpack-defaults